### PR TITLE
Better documentation on watch and babel

### PR DESCRIPTION
**Pull request by gabrielpoca**
Why:

Watch was updated so it would also watch files. Babel was update to have stage
0 by default.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,6 +304,8 @@ Inject file references into index.html if the files with the extension `.css` or
 </html>
 ```
 
+If `watch` is enable this files will be watched for changes, and copied again on change.
```
Comment (azevedo) [inline README.md]: Typos:
**enabled**
**these**

File: README.md [hunk 2]
```diff
@@ -325,15 +327,16 @@ Enable HTML5 pushState. When this option is enabled, styleguide will use history
 Type: `Object`  
 Default: `null`
 
-A usage example is below. See the [babel docs](http://babeljs.io/docs/usage/options/) for the complete list.
+Configuration object for babel. Take the followin as an aexample.
```
Comment (azevedo) [inline README.md]: Typos:
**following**
**example**

File: README.md [hunk 3]
```diff
@@ -325,15 +327,16 @@ Enable HTML5 pushState. When this option is enabled, styleguide will use history
...
 
 ``` js
 {
   babelConfig: {
-    stage: 0
+    stage: 1
   }
 }
 ```
+The `stage: 0` is enabled by default. See the [babel docs](http://babeljs.io/docs/usage/options/) for the complete list.
```
Comment (azevedo) [inline README.md]: I'd remove the first two `the`s :)

**Comment by gabrielpoca** [comment]
@azevedo-252 fixed

**Comment by azevedo** [comment]
:+1:

**Pull request merged**